### ISSUE TITLE
How to extract tables without headers

### ISSUE THREAD

**Issue by sHAsHiLx** [kedarvj/mysqldumpsplitter#8]
Having dump without any headers, just pure data, so dump looks like

INSERT INTO `members10` VALUES (...
INSERT INTO `members11` VALUES (...
INSERT INTO `members12` VALUES (...
INSERT INTO `members13` VALUES (...

as I see when I choose option "--match_str members10" it search for "-- Table structure for table `members10`" but it's not in dump. Dump is just pure data without database structure. Structure created from other dump. File is 100gb sized and I need to split it in files members10 members11 members12 etc.

**Comment by kedarvj** [comment]
Right. Script doesn't work well with sql files created with --no-create-info. Will look to improve.
